### PR TITLE
 Refine Language and Inclusivity in MtF.wiki Introduction.

### PR DESCRIPTION
**Pull request by ly-nld**

### CODE REVIEW

**Comment by ly-nld** [comment]
Please let me know if these modifications are sufficient 😊

**Review by saeziae — APPROVED** [review]
lgtm :)

File: README-EN.md [hunk 1]
```diff
@@ -11,17 +11,17 @@ Brother Site: [FtM.wiki](https://ftm.wiki)
 [![GitHub Watchers][badge-gh-watch]][repo]
 [![GitHub Stars][badge-gh-stars]][repo]
 
-This site was founded initially for China mainland's transgenders. However, if you would like to contribute information for the transgenders in your region, feel free to do so.
+This site was founded initially for mainland China's transgender people. However, if you would like to contribute information for transgender individuals in your region, feel free to do so.
```
Comment (sakurakouji-aya) [inline README-EN.md:14]: for the transgender individuals ..., please feel free ....

File: README-EN.md [hunk 2]
```diff
@@ -11,17 +11,17 @@ Brother Site: [FtM.wiki](https://ftm.wiki)
...
 
 ## Content
 
 Welcome to the MtF.wiki. This is a place that may provide help for some transgender women.
 
-MtF (Male-to-Female), or transgender female, refers to a person who was assigned male at birth but identifies as female.
+MtF (Male-to-Female) or transgender female refers to a person who was assigned male at birth but is a woman.
 
-MtF.wiki is committed to becoming a free, open, and continuously updated transgender knowledge integration site, where you can learn about all kinds of medical information and dailylife guides related to transgender.
+MtF.wiki is dedicated to evolving into a free, open, and consistently updated platform for the integration of transgender knowledge, offering insights into various medical information and daily life guides relevant to transgender individuals and their experiences.
```
Comment (sakurakouji-aya) [inline README-EN.md:22]: ... daily life guidance ...

File: README-EN.md [hunk 3]
```diff
@@ -11,17 +11,17 @@ Brother Site: [FtM.wiki](https://ftm.wiki)
...
 
-The MtF.wiki is currently in its early stages of development. Various materials may have not been included, and there may even be some errors. All enthusiastic sisters are welcomed to join us and contribute to the MtF.wiki project.
+The MtF.wiki is currently in its early stages of development. Various resources may have not been included, and there may even be some errors. All enthusiastic sisters are welcome to join us and contribute to the MtF.wiki project.
```
Comment (sakurakouji-aya) [inline README-EN.md:24]: All enthusiastic contributors ... and commit to the ....

File: content/en/about.md [hunk 4]
```diff
@@ -16,13 +16,13 @@ Project <i class="bi bi-github" aria-label="GitHub"></i> GitHub Repo: [project-t
 
 Brother site: <https://FtM.wiki>
 
-Thanks goes to these GitHub contributors:
+Gratitude goes to these GitHub contributors:
```
Comment (sakurakouji-aya) [inline content/en/about.md:19]: My deepest thanks goes to ...

File: content/en/docs/hrt/singapore/singapore-hrt-overview.md [hunk 5]
```diff
@@ -11,14 +11,14 @@ The main place to get HRT is [Changi General Hospital](https://www.cgh.com.sg/).
 
 The process consists of two steps:
 
-1. Psychiatrist visit (gender disphoria diagnosis)
-1. Endocrinology visit (obtain HRT)
+1. Psychiatrist visit (for a gender dysphoria diagnosis)
+1. Endocrinology visit (for obtaining HRT)
```
Comment (RimuroSuraimu) [inline content/en/docs/hrt/singapore/singapore-hrt-overview.md:15]: for obtaining HRT perscription

**Review by RimuroSuraimu — APPROVED** [review]
think is ok to go

File: content/en/docs/_index.md [hunk 6]
```diff
@@ -8,12 +8,12 @@ draft: false
 Welcome to the {{< mtf-wiki >}}, a place that may help some transgender women.
 
 {{< notice info >}}
-Thought this site was founded initially for Chinese trans, we provide this English page to show our visibility. However, if you would like to contribute information for trans in your region, welcome to do it.
+Though this site was founded initially for Chinese trans, we provide this English page to show our visibility. However, if you would like to contribute information for trans in your region, welcome to do it.
```
Comment (sakurakouji-aya) [inline content/en/docs/_index.md:11]: ... for Chinese transgender people ... for transgender people in your region, it is always welcome to do it.

File: content/en/docs/_index.md [hunk 7]
```diff
@@ -8,12 +8,12 @@ draft: false
...
 {{< /notice >}}
 
-MtF (Male-to-Female), transgender female, refers to a female whose sex[^1] is male when she was born.
+MtF (Male-to-Female), or transgender female, refers to a female whose gender[^1] was assigned male at birth but is a woman.
 
-{{< mtf-wiki >}} is committed to becoming a free, open and continuously updated transgender knowledge integration site, where you can learn about all kinds of medical information and dailylife guides related to transgender.
+{{< mtf-wiki >}} is dedicated to evolving into a free, open, and consistently updated platform for the integration of transgender knowledge, offering insights into various medical information and daily life guides relevant to transgender individuals and their experiences.
```
Comment (sakurakouji-aya) [inline content/en/docs/_index.md:16]: Mtf.wiki is committed to becoming a free, open, and constantly updated platform for the integration of transgender knowledge, providing insights into diverse medical information and everyday living guidelines important to transgender individuals and their experiences.

File: README-EN.md [hunk 8]
```diff
@@ -11,17 +11,17 @@ Brother Site: [FtM.wiki](https://ftm.wiki)
...
 
 ## Content
 
 Welcome to the MtF.wiki. This is a place that may provide help for some transgender women.
 
-MtF (Male-to-Female), or transgender female, refers to a person who was assigned male at birth but identifies as female.
+MtF (Male-to-Female) or transgender female refers to a person who was assigned male at birth but is a woman.
 
-MtF.wiki is committed to becoming a free, open, and continuously updated transgender knowledge integration site, where you can learn about all kinds of medical information and dailylife guides related to transgender.
+MtF.wiki is dedicated to evolving into a free, open, and consistently updated platform for the integration of transgender knowledge, offering insights into various medical information and daily life guidance relevant to transgender individuals and their experiences.
```
Comment (sakurakouji-aya) [inline README-EN.md:22]: ... constantly updated platform ...  everyday living guidelines important ...

**Review by sakurakouji-aya — COMMENTED** [review]
These are the changes.

File: content/en/about.md (hunk 4 above)
Comment (ly-nld) [reply content/en/about.md]: "goes" isn't grammatically correct in this sentence, the correct wording is "go" when referring to multiple people.

**Comment by ly-nld** [comment]
I think I have implemented all the suggestions. Sorry that I missed those 😅

**Review by sakurakouji-aya — APPROVED** [review]

**Pull request merged**